api/db/migrations/20240603182838_populate_organization_preferences/migration.sql

```diff
@@ -0,0 +1,30 @@
+BEGIN;
+
+CREATE OR REPLACE FUNCTION populate_organization_preferences()
+RETURNS void AS $$
+DECLARE 
+    current_reporting_period_id INT;
+BEGIN 
+    -- Get one valid reporting period id --
+    SELECT id INTO current_reporting_period_id 
```

Comment:
Does this need to be ordered?

Reply:
Good q! Not really -- this shouldn't touch production data, just run a one-off migration to fix data that's broken in staging -- so it's sufficient to assign it to any reporting period ATM